### DISCUSSION TITLE
React Hook Form validations are not working with MUI Radiogroup

### DISCUSSION THREAD

**Discussion by jaykeerti** [react-hook-form/react-hook-form#11484] [General]
I am able to validate the input box/Text Field. i am unable to validate RadioGroup is selected or not. Please let me know what mistake i am doing here.

I need a functionality in which no radio buttons are selected and user tries to submit, it will throw an error stating that field is required

Here is my CSB
https://codesandbox.io/p/sandbox/modest-marco-xzjl32

**Comment by Moshyfawn** [comment]
You never use `field.onChange` or any other handlers to pass your `RadioGroup` value to RHF for validation.

**Reply by jaykeerti** [reply]
@Moshyfawn My use case here is when the user does not select any radio option i want to throw an error saying it is required using the [fieldState ](https://react-hook-form.com/docs/usecontroller/controller) object.

Here is a working example for TextField. Try to submit the form with an empty TextField. It will throw the validation error
https://codesandbox.io/p/sandbox/modest-marco-forked-mdj7hw
